web/tools/webpack.config.babel.js

```diff
@@ -63,6 +64,7 @@ const createConfig = (env = {}) => {
   const wwwDirectory = path.resolve(__dirname, '../www')
   const distDirectory = path.resolve(__dirname, `../dist/${buildConfigName}`)
   const srcDirectory = path.resolve(__dirname, '../src')
+  const bundleReportDirectory = path.resolve(__dirname, '../reports/bundle')
```

Comment:
This is written to `web/reports/bundle`. Other reports are written to the rootDir, aren't they? Should we make this consistent, i.e. only rootdir/reports or rootDir/web/reports and rootDir/native/reports?

Reply:
Nothing is written to the root directory of the project. Reports are in web/reports and native/reports